### PR TITLE
Setup feature for mobile version

### PR DESCRIPTION
**Pull request by M-AminAlizadeh**
The things I did in this pull request are:
1. Build ``works`` and ``about myself`` components with Html / CSS 
2. Use Grid and Flexbox layout
3. Use linterns to prevent semantic errors
4. Use Gitflow and branching
5. Documented my workflow in the Readme file
6. Use Figma design to implement the values of elements and components
7. Add interaction to buttons using pseudo-classes
8. Set a proper name for the branch and commit the changes by explaining the details

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,5 +71,267 @@ <h1 class="headline-greeting">I’m Amin.<br/>
             </div>
         </div>
     </section>
+    <!-- Works -->
```
Comment (souad988) [inline index.html:74]: You have done a great work , one very small detail you missed is the background color of this section Please do all you can to replicate figma design 

**figma design**
temp1 figme background

**your design**
temp1 student background

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @M-AminAlizadeh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔Professional PR description
✔Very good commit messages
✔Professional README
✔Good use of html tags

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -37,16 +38,17 @@ This is my Portfolio repository
...
+- **Layout with Flexbox & Grid**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://frabjous-melomakarona-10efb2.netlify.app)
+- [Live Demo](https://coruscating-maamoul-2ff2cb.netlify.app/)
```
Comment (braincee) [inline README.md:51]: - [ ] [OPTIONAL] I kindly suggest updating the live demo link or removing it. 😄

**Review by braincee — APPROVED** [review]
Hi @M-AminAlizadeh  , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ] [OPTIONAL] I suggest adding an author section to the README file which includes links to social media handles. Doing so improves your connections with other software developers and recruiters. 😄 

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 2 above)
Comment (M-AminAlizadeh) [reply README.md:51]: Sure I will

**Pull request merged**